### DISCUSSION TITLE
Website Build

### DISCUSSION THREAD

**Discussion by mudit023** [mudit023/templates#84] [Ideas]
This space is to discuss the website build mentioned in issue #82 

Share your ideas and progress here👇

**Comment by MOHAMED-EHAB-DEV** [comment]
We can Use A Simple Design with a Simple Header and A Simple Footer and some divs in the middle @mudit023

**Reply by mudit023** [reply]
Okay. You can start with that.

**Reply by MOHAMED-EHAB-DEV** [reply]
> Okay. You can start with that.

Where we will put our code 🙂?

**Reply by mudit023** [reply]
Can we create a separate folder for it and keep our website code there?

**Reply by mudit023** [reply]
Or is there any better way?

**Reply by MOHAMED-EHAB-DEV** [reply]
> Or is there any better way?

Yes, there are many ways

We must create two saparated repositories and the way to do that is to create a new organization

**Reply by mudit023** [reply]
Created an organization **Templates**. Have sent you the invite.

**Reply by MOHAMED-EHAB-DEV** [reply]
> Created an organization **Templates**. Have sent you the invite.

Please Resend the invite and send the link to me or change the own of this repository to the organization

**Reply by MOHAMED-EHAB-DEV** [reply]
> Created an organization **Templates**. Have sent you the invite.

Hey Mudit 

What Language or Framework will we use ?
React, Html CSS JS or What ?

**Reply by mudit023** [reply]
What do you suggest? We are open to ideas.

**Reply by MOHAMED-EHAB-DEV** [reply]
> What do you suggest? We are open to ideas.

Ok, I will try the both

**Reply by MOHAMED-EHAB-DEV** [reply]
> Created an organization **Templates**. Have sent you the invite.

I Joined the Organization

**Reply by mudit023** [reply]
> > Created an organization **Templates**. Have sent you the invite.
> 
> I Joined the Organization

Awesome! You have the right to add the repo to the organization and make changes to it. You can start with your idea.

**Reply by MOHAMED-EHAB-DEV** [reply]
> > > Created an organization **Templates**. Have sent you the invite.
> > 
> > 
> > I Joined the Organization
> 
> Awesome! You have the right to add the repo to the organization and make changes to it. You can start with your idea.

Mudit,  Are you here ?

**Reply by MOHAMED-EHAB-DEV** [reply]
> > > Created an organization **Templates**. Have sent you the invite.
> > 
> > 
> > I Joined the Organization
> 
> Awesome! You have the right to add the repo to the organization and make changes to it. You can start with your idea.

Hey Sir

I tried to fetch GitHub API but there is a problem

We want the image or logo of the creator of the template and i see that it is an string when i fetch it so we must create add the template by ourselfs + We want to create a logo for our templates organization + transfer the owner ship of this repo into Our Organization

Make me an administrator in the Organization 

Are you Ready ?

**Reply by MOHAMED-EHAB-DEV** [reply]
@mudit023

**Reply by mudit023** [reply]
I also thought to transfer the ownership to the organization. But I don't know about its consequences, so I decided not to do that now. Let's wait for the hacktoberfest to get over, as right now there is traffic and I don't want, that it gets harder for people to contribute.

We'll start working on the website after October.

**Reply by MOHAMED-EHAB-DEV** [reply]
> I also thought to transfer the ownership to the organization. But I don't know about its consequences, so I decided not to do that now. Let's wait for the hacktoberfest to get over, as right now there is traffic and I don't want, that it gets harder for people to contribute.
> 
> We'll start working on the website after October.

Ok, But we must transfer the ownership

When we transfer the owner ship to the organization this repository link will redirect to organization link as reactplay repository his owner create an organization and transfer the ownership when we go to [Templates Repository](https://github.com/mudit023/templates) it will redirect to Templates Repository in Our Organization

I prefer now to transfer the ownership of this repository

**Comment by MOHAMED-EHAB-DEV** [comment]
Hi @mudit023 

I have some prefer updates to our organization:
1. Make A Logo to Our Organization
2. Change The name of the organization from **Tempates** to **Templates**

I will take up The Logo if you want

**Reply by mudit023** [reply]
Hey, I'm thinking about a few options regarding the organization. I don't want to take any decision as of now. Let's keep the website build on hold for some time.

**Reply by MOHAMED-EHAB-DEV** [reply]
> Hey, I'm thinking about a few options regarding the organization. I don't want to take any decision as of now. Let's keep the website build on hold for some time.

Ok
